### ISSUE TITLE
Successfully installed Simba but cannot run Simba

### ISSUE THREAD

**Issue by flyegg** [sgoldenlab/simba#62]
Hi,

I created a conda environment for Simba only and installed Simba under this environment. The installation was smooth but simba keep deliver error message 

I have tried to run DLC in a different environment and everything worked perfectly. I guess the hardware should not be a problem here. 

Here are the scripts and error messages I got. Basically I did three thing: uninstall simba, reinstall simba and run simba. 

Is there any clue?

Best regards

-Yi

(Simba-GPU) yi@Precision-5820-Tower:~$ pip uninstall simba-uw-tf 
Found existing installation: Simba-UW-tf 1.2.19
Uninstalling Simba-UW-tf-1.2.19:
  Would remove:
    /home/yi/anaconda3/envs/Simba-GPU/bin/simba
    /home/yi/anaconda3/envs/Simba-GPU/lib/python3.6/site-packages/Simba_UW_tf-1.2.19.dist-info/*
    /home/yi/anaconda3/envs/Simba-GPU/lib/python3.6/site-packages/simba/*
Proceed (y/n)? y
  Successfully uninstalled Simba-UW-tf-1.2.19
(Simba-GPU) yi@Precision-5820-Tower:~$ pip uninstall simba-uw-tf
WARNING: Skipping simba-uw-tf as it is not installed.
(Simba-GPU) yi@Precision-5820-Tower:~$ pip install simba-uw-tf
Collecting simba-uw-tf
  Using cached Simba_UW_tf-1.2.19-py3-none-any.whl (3.3 MB)
Requirement already satisfied: tqdm==4.30.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (4.30.0)
Requirement already satisfied: Shapely==1.7 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (1.7.0)
Requirement already satisfied: imutils==0.5.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.5.2)
Requirement already satisfied: pyyaml==5.3.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (5.3.1)
Requirement already satisfied: dtreeviz==0.8.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.8.1)
Requirement already satisfied: wxpython==4.0.4 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (4.0.4)
Requirement already satisfied: tensorflow-gpu==1.14.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (1.14.0)
Requirement already satisfied: tabulate==0.8.3 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.8.3)
Requirement already satisfied: scipy==1.1.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (1.1.0)
Requirement already satisfied: numpy==1.18.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (1.18.1)
Requirement already satisfied: deeplabcut==2.0.9 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (2.0.9)
Requirement already satisfied: protobuf==3.6.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (3.6.0)
Requirement already satisfied: deepposekit==0.3.5 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.3.5)
Requirement already satisfied: imblearn==0.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.0)
Requirement already satisfied: scikit-learn==0.22.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.22.2)
Requirement already satisfied: seaborn==0.9.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.9.0)
Requirement already satisfied: xgboost==0.90 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.90)
Requirement already satisfied: matplotlib==3.0.3 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (3.0.3)
Requirement already satisfied: yellowbrick==0.9.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.9.1)
Requirement already satisfied: eli5==0.10.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.10.1)
Requirement already satisfied: Pillow==5.4.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (5.4.1)
Requirement already satisfied: opencv-python==3.4.5.20 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (3.4.5.20)
Requirement already satisfied: pandas==0.25.3 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.25.3)
Requirement already satisfied: scikit-image==0.14.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.14.2)
Requirement already satisfied: graphviz==0.11 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.11)
Requirement already satisfied: imgaug==0.4.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from simba-uw-tf) (0.4.0)
Requirement already satisfied: colour in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from dtreeviz==0.8.1->simba-uw-tf) (0.1.5)
Requirement already satisfied: six in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from wxpython==4.0.4->simba-uw-tf) (1.11.0)
Requirement already satisfied: keras-preprocessing>=1.0.5 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (1.1.2)
Requirement already satisfied: astor>=0.6.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (0.8.1)
Requirement already satisfied: absl-py>=0.7.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (0.10.0)
Requirement already satisfied: google-pasta>=0.1.6 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (0.2.0)
Requirement already satisfied: tensorboard<1.15.0,>=1.14.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (1.14.0)
Requirement already satisfied: tensorflow-estimator<1.15.0rc0,>=1.14.0rc0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (1.14.0)
Requirement already satisfied: wheel>=0.26 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (0.31.1)
Requirement already satisfied: grpcio>=1.8.6 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (1.31.0)
Requirement already satisfied: termcolor>=1.1.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (1.1.0)
Requirement already satisfied: keras-applications>=1.0.6 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (1.0.8)
Requirement already satisfied: wrapt>=1.11.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (1.12.1)
Requirement already satisfied: gast>=0.2.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorflow-gpu==1.14.0->simba-uw-tf) (0.4.0)
Requirement already satisfied: tensorpack~=0.9.7.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (0.9.7.1)
Requirement already satisfied: python-dateutil~=2.7.3 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (2.7.5)
Requirement already satisfied: click in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (7.1.2)
Requirement already satisfied: h5py~=2.7 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (2.10.0)
Requirement already satisfied: statsmodels~=0.9.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (0.9.0)
Requirement already satisfied: intel-openmp in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (2020.0.133)
Requirement already satisfied: moviepy~=0.2.3.5 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (0.2.3.5)
Requirement already satisfied: patsy in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (0.5.1)
Requirement already satisfied: certifi in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (2020.6.20)
Requirement already satisfied: imageio~=2.3.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (2.3.0)
Requirement already satisfied: chardet~=3.0.4 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (3.0.4)
Requirement already satisfied: easydict~=1.7 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (1.9)
Requirement already satisfied: setuptools in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (49.6.0.post20200814)
Requirement already satisfied: ipython-genutils~=0.2.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (0.2.0)
Requirement already satisfied: tables in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (3.6.1)
Requirement already satisfied: ipython~=6.0.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (6.0.0)
Requirement already satisfied: requests in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (2.24.0)
Requirement already satisfied: ruamel.yaml~=0.15 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from deeplabcut==2.0.9->simba-uw-tf) (0.16.10)
Requirement already satisfied: imbalanced-learn in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from imblearn==0.0->simba-uw-tf) (0.7.0)
Requirement already satisfied: joblib>=0.11 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from scikit-learn==0.22.2->simba-uw-tf) (0.16.0)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from matplotlib==3.0.3->simba-uw-tf) (2.4.7)
Requirement already satisfied: cycler>=0.10 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from matplotlib==3.0.3->simba-uw-tf) (0.10.0)
Requirement already satisfied: kiwisolver>=1.0.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from matplotlib==3.0.3->simba-uw-tf) (1.2.0)
Requirement already satisfied: jinja2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from eli5==0.10.1->simba-uw-tf) (2.11.2)
Requirement already satisfied: attrs>16.0.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from eli5==0.10.1->simba-uw-tf) (19.3.0)
Requirement already satisfied: pytz>=2017.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from pandas==0.25.3->simba-uw-tf) (2020.1)
Requirement already satisfied: PyWavelets>=0.4.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from scikit-image==0.14.2->simba-uw-tf) (1.1.1)
Requirement already satisfied: cloudpickle>=0.2.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from scikit-image==0.14.2->simba-uw-tf) (1.5.0)
Requirement already satisfied: networkx>=1.8 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from scikit-image==0.14.2->simba-uw-tf) (2.4)
Requirement already satisfied: dask[array]>=1.0.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from scikit-image==0.14.2->simba-uw-tf) (2.23.0)
Requirement already satisfied: werkzeug>=0.11.15 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorboard<1.15.0,>=1.14.0->tensorflow-gpu==1.14.0->simba-uw-tf) (1.0.1)
Requirement already satisfied: markdown>=2.6.8 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorboard<1.15.0,>=1.14.0->tensorflow-gpu==1.14.0->simba-uw-tf) (3.2.2)
Requirement already satisfied: pyzmq>=16 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorpack~=0.9.7.1->deeplabcut==2.0.9->simba-uw-tf) (19.0.2)
Requirement already satisfied: msgpack>=0.5.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorpack~=0.9.7.1->deeplabcut==2.0.9->simba-uw-tf) (1.0.0)
Requirement already satisfied: msgpack-numpy>=0.4.4.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorpack~=0.9.7.1->deeplabcut==2.0.9->simba-uw-tf) (0.4.6.1)
Requirement already satisfied: psutil>=5 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tensorpack~=0.9.7.1->deeplabcut==2.0.9->simba-uw-tf) (5.7.2)
Requirement already satisfied: decorator<5.0,>=4.0.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from moviepy~=0.2.3.5->deeplabcut==2.0.9->simba-uw-tf) (4.4.2)
Requirement already satisfied: numexpr>=2.6.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from tables->deeplabcut==2.0.9->simba-uw-tf) (2.7.1)
Requirement already satisfied: jedi>=0.10 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (0.17.2)
Requirement already satisfied: prompt-toolkit<2.0.0,>=1.0.4 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (1.0.18)
Requirement already satisfied: pickleshare in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (0.7.5)
Requirement already satisfied: traitlets>=4.2 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (4.3.3)
Requirement already satisfied: simplegeneric>0.8 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (0.8.1)
Requirement already satisfied: pygments in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (2.6.1)
Requirement already satisfied: pexpect; sys_platform != "win32" in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (4.8.0)
Requirement already satisfied: idna<3,>=2.5 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from requests->deeplabcut==2.0.9->simba-uw-tf) (2.10)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from requests->deeplabcut==2.0.9->simba-uw-tf) (1.25.10)
Requirement already satisfied: ruamel.yaml.clib>=0.1.2; platform_python_implementation == "CPython" and python_version < "3.9" in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from ruamel.yaml~=0.15->deeplabcut==2.0.9->simba-uw-tf) (0.2.0)
Requirement already satisfied: MarkupSafe>=0.23 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from jinja2->eli5==0.10.1->simba-uw-tf) (1.1.1)
Requirement already satisfied: toolz>=0.8.2; extra == "array" in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from dask[array]>=1.0.0->scikit-image==0.14.2->simba-uw-tf) (0.10.0)
Requirement already satisfied: importlib-metadata; python_version < "3.8" in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from markdown>=2.6.8->tensorboard<1.15.0,>=1.14.0->tensorflow-gpu==1.14.0->simba-uw-tf) (1.7.0)
Requirement already satisfied: parso<0.8.0,>=0.7.0 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from jedi>=0.10->ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (0.7.1)
Requirement already satisfied: wcwidth in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from prompt-toolkit<2.0.0,>=1.0.4->ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (0.2.5)
Requirement already satisfied: ptyprocess>=0.5 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from pexpect; sys_platform != "win32"->ipython~=6.0.0->deeplabcut==2.0.9->simba-uw-tf) (0.6.0)
Requirement already satisfied: zipp>=0.5 in ./anaconda3/envs/Simba-GPU/lib/python3.6/site-packages (from importlib-metadata; python_version < "3.8"->markdown>=2.6.8->tensorboard<1.15.0,>=1.14.0->tensorflow-gpu==1.14.0->simba-uw-tf) (3.1.0)
Installing collected packages: simba-uw-tf
Successfully installed simba-uw-tf-1.2.19
(Simba-GPU) yi@Precision-5820-Tower:~$ simba
Traceback (most recent call last):
  File "/home/yi/anaconda3/envs/Simba-GPU/bin/simba", line 8, in <module>
    sys.exit(main())
  File "/home/yi/anaconda3/envs/Simba-GPU/lib/python3.6/site-packages/simba/SimBA.py", line 5217, in main
    app = SplashScreen(root)
  File "/home/yi/anaconda3/envs/Simba-GPU/lib/python3.6/site-packages/simba/SimBA.py", line 5197, in __init__
    self.Splash()
  File "/home/yi/anaconda3/envs/Simba-GPU/lib/python3.6/site-packages/simba/SimBA.py", line 5202, in Splash
    self.image = Image.open(os.path.join(scriptdir,"TheGoldenLab.png"))
  File "/home/yi/anaconda3/envs/Simba-GPU/lib/python3.6/site-packages/PIL/Image.py", line 2634, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/home/yi/anaconda3/envs/Simba-GPU/lib/python3.6/site-packages/simba/TheGoldenLab.png'

**Comment by benjaws** [comment]
Hi @flyegg , check the filename of the PNG, in my case there was an error.
